<launch>
  <node name="bottomCamera" pkg="pauvsi_m7" type="cameraDriver" output="screen">
  	<param name="frame_rate" value="30" />
  	<param name="serial_number" value="15374828" />
  	<param name="crop" value="true" />
  	<param name="camera_position" value="bottomCam" />
  	<param name="crop_x" value="1000" />
  	<param name="crop_y" value="1000" />
  </node>
  
  <node name="CameraCalibrator" pkg="pauvsi_m7" type="cameraCalibrator" output="screen">
  	<param name="cameraTopic" value="/PGCameraDriver/bottomCam/color/undistorted" />
  </node>
</launch>
